### ISSUE TITLE
In model.fit, have each metric given a new line in display. 

### ISSUE THREAD

**Issue by Santosh-Gupta** [tensorflow/tensorflow#40630]
**Describe the feature and the current behavior/state.**

Instead of having each metric on one line in the display for model.fit, have each metric on a new line.

**Who will benefit with this feature?**

Those who are trying to monitor a ton of metrics. In my particular case, I am trying to track down the cause of exploding gradients, so I'm monitoring the norm of each trainable_weight. This makes for a very long line to side-scroll through.

**Comment by jpodivin** [comment]
@Santosh-Gupta 
You might want to consider alternative ways to monitor learning process. Depending on your needs you could save the return value of the fit method ('history') as a csv. Or just passing to plotting method of your choice. You could also use tensorboard to plot the training metrics and analyze the weights.
